optimism2/aave/interest.sql

```diff
@@ -0,0 +1,113 @@
+CREATE TABLE IF NOT EXISTS aave.interest (   
+    day timestamptz,
```

Comment:
might be worth having a version here as well despite optimism only having v3

Reply:
Agree, I've added this for Optimism aswell